app/components/publications/publication/case/info/publication-case-info-panel.hbs

```diff
@@ -137,7 +137,7 @@
                   {{else}}
                     <Auk::Datepicker
                       @date={{@publicationFlow.agendaItemTreatment.startDate}}
-                      @onChange={{this.setDecisionDate}}
+                      @onChange={{set this "agendaItemTreatment.startDate"}}
```

Comment:
Deze PR is niet de oorzaak, maar bij `@date` kunnen we dan ook best `this.agendaItemTreatment.startDate` gebruiken ipv `@publicationFlow.agendaItemTreatment.startDate`

Reply:
Bij set ook iets logischer om `{{set this.agendaItemTreatment "startDate"}}` te gebruiken